metadata-integration/java/acryl-spark-lineage/build.gradle

```diff
@@ -106,6 +108,8 @@ shadowJar {
     exclude(dependency {
       exclude_modules.contains(it.name)
     })
+    exclude(dependency("org.slf4j::"))
```

Comment:
I don't think this is correct. I think its supposed to be `dependency("org.slf4j::*")`

Reply:
Actually, it works but I will change to yours as it is more expressive

Reply:
I tested and yours doesn't work but this one works.